notebooks/223-gpt2-text-prediction/223-gpt2-text-prediction.ipynb

```diff
@@ -10,39 +10,13 @@
    "source": [
```

Comment:
<p>Why no more dynamic shapes?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>model exported to onnx with dynamic shape and converted by mo with input_range, so no need additionally reshape it to get dynamic shape</p>